index.js

```diff
@@ -32,7 +32,7 @@ const plugins = [];
 /**
  * Yarn Plug'n'Play means that dependency hoisting doesn't work like it normally
  * would with the standard `node_modules` configuration. This is by design, as
- * hoisting is unpredicted.
+ * hoisting is unpredictable.
```

Comment:
is hoisting supposed to say hosting?

Reply:
No, hoisting is the process of pulling dependencies from a package you use up into the top-most scope.

In this case, it would be by pulling our webpack config dependencies up to our core JS package, for example.